action.yaml

```diff
@@ -61,6 +61,10 @@ inputs:
     description: 'filter vulnerabilities with OPA rego language'
     required: false
     default: ''
+  hide-progress:
+    description: 'hide progress output'
+    required: false
+    default: 'true'
```

Comment:
Looks like the `--no-progress` flag is not available for `config` scan type. Should we default this to false?

Reply:
The --no-progress Trivy cli switch was not supported when using the config scan type, but it was included by default in the command line by this action regardless of parameters. This effectively made the default to 'true'. As part of this PR i moved it behind a parameter called 'hide-progress' so it could be turned off when running the config scan type, but still configurable when using other scan types. I left the default as true because that was effectively it's state prior to me adding the parameter for it.